Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md

```diff
@@ -6,6 +6,17 @@ versionFrom: 7.0.0
 
 All new projects on Umbraco Cloud are automatically protected by Umbraco Latch. This means, that the default Umbraco Cloud URL for your project as well as any new hostnames you add will be assigned a TLS certificate automatically.
 
+:::note
+Certificates last 3 months and are automatically renewed, as long as the prerequisites below are met.
+:::
+
+## Prerequisites
+In order for Latch to work and the certificate to be applied the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.
+
+Latch needs the site to have a status of 200 in order for the certificate to be applied and renewed. This means that every 3 months Latch will go through the renewal process and the prerequisites have to be met each time in order for it to work.
+
+If you wish to have a custom 404 page on the root of your website then you will need to purchase and apply a custom certificate instead.
```

Comment:
Let's then add a link to the manual certificates article - what do you think?

```suggestion
If you wish to have a custom 404 page on the root of your website then you will need to purchase and apply a custom certificate instead.
```

Reply:
Yeah, sounds like a good idea. Can you add it there? I'm unsure on how to do is partway through the PR 😐

Reply:
I believe the path is `../Manage-Hostnames/Security-Certificates/`.
Can add it I you like. Let me know!